### ISSUE TITLE
[0.37.0/ppc64el] segmentation fault during test

### ISSUE THREAD

**Issue by cdluminate** [numba/numba#3050]
https://buildd.debian.org/status/fetch.php?pkg=numba&arch=ppc64el&ver=0.37.0-1&stamp=1521778678&raw=0

```
numba/tests/test_array_manipulation.py::TestArrayManipulation::test_add_axis2 PASSED [  1%]
numba/tests/test_array_manipulation.py::TestArrayManipulation::test_add_axis2_npm PASSED [  1%]
numba/tests/test_array_manipulation.py::TestArrayManipulation::test_array_reshape Segmentation fault
E: pybuild pybuild:330: test: plugin custom failed with: exit code=139: cd /<<PKGBUILDDIR>>/.pybuild/cpython2_2.7_numba/build && python2.7 -Wd -m pytest numba/tests -v -rs
dh_auto_test: pybuild --test --test-pytest -i python{version} -p 2.7 returned exit code 13
make[1]: Leaving directory '/<<PKGBUILDDIR>>'
```

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=863511

**Comment by obilaniu** [comment]
For PPC64, s390x, ARM and MIPS, please see the resolution of #2451 , and the release notes of [v0.38.0](https://github.com/numba/numba/blob/master/CHANGE_LOG).

In particular, do note that the segfault issues on non-x86 architectures are solved with LLVM **6.0.1**, llvmlite 0.23 and Numba 0.38 or above. LLVM 6.0.0 will **not** work. LLVM 6.0.1 is **not** yet released; However, `6.0.1-rc2` has the required fixes, and 6.0.1's final release is imminent.

**Comment by stuartarchibald** [comment]
Closing this issue as no new information has been provided. It is assumed that the above resolved the reported issue, if this is not the case please reopen the issue with new information. Many thanks.
